src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx

```diff
@@ -101,14 +101,14 @@ class SpanDetail extends React.Component<Props, State> {
       field: ['transaction', 'id', 'trace.span'],
       sort: ['-id'],
       query: `event.type:transaction trace:${traceID} trace.parent_span:${spanID}`,
-      project: organization.features.includes('global-views')
+      project: (organization.features.includes('global-views')
         ? [ALL_ACCESS_PROJECTS]
-        : [Number(event.projectID)],
+        : [Number(event.projectID)]) as number[] | undefined,
       start,
       end,
     };
 
-    if (query.project.length === 0) {
+    if (query.project!.length === 0) {
```

Comment:
will be `project` always available? I would avoid using `!`

Reply:
Reading this, `query.project.length` is always non-zero.

Reply:
Will be deleted in https://github.com/getsentry/sentry/pull/20709